### PR TITLE
TMP36 sensor ported

### PR DESCRIPTION
**Pull request by dshynkev**

### CODE REVIEW

**Comment by dshynkev** [comment]
Wiring diagram:
temperature_tmp36fritz

File: tmp36-temperature-sensor/include/TemperatureSensor.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+#include <AnalogPinReader.h>
+
+namespace wlp {
+    namespace TMP36 {
+        class TemperatureSensor: public AnalogPinReader {
+        public:
+            explicit TemperatureSensor(Board::AnalogPin analog_pin)
+            : AnalogPinReader(analog_pin) {};
+            virtual double read_value() override;
```
Comment (Mogball) [inline tmp36-temperature-sensor/include/TemperatureSensor.h:12]: does this need to be virtual?

Comment (Mogball) [reply tmp36-temperature-sensor/include/TemperatureSensor.h:12]: also realised some of the other functions for the other sensors are like this too

Comment (dshynkev) [reply tmp36-temperature-sensor/include/TemperatureSensor.h:12]: You did ask me to make it virtual in #2 and #3

Comment (Mogball) [reply tmp36-temperature-sensor/include/TemperatureSensor.h:12]: I meant that the parent function be virtual, so that you can mark these as override

Comment (dshynkev) [reply tmp36-temperature-sensor/include/TemperatureSensor.h:12]: They would still be virtual: https://stackoverflow.com/a/1404843

Comment (dshynkev) [reply tmp36-temperature-sensor/include/TemperatureSensor.h:12]: I thought it was an error to `override` a `virtual` with non-`virtual`, but apparently it just makes both methods virtual.

Comment (Mogball) [reply tmp36-temperature-sensor/include/TemperatureSensor.h:12]: usually a function is marked `virtual` when it's meant to be overridden (more of a code readability issue) so unless you plan to override `read_value` for the sensors `virtual` should be removed

I'll merge this now and fix after

**Pull request merged**